server/app/vectors.py

```diff
@@ -22,7 +23,42 @@ def vector_to_world(vector: schemas.VectorInDB) -> schemas.VectorOut:
     Returns:
         schemas.VectorOut: Vector sent over the wire.
     """
-    return schemas.VectorOut(**vector.dict(), sequence_length=len(vector.sequence))
+    vec_in_db_dict = vector.dict()
+
+    if len(vector.children) > 0:
```

Comment:
Just don't check this condition and avoid the `else` duplication.

Reply:
You're right! This was sloppy coding on my part! Apologies for that!